neuraxle/metaopt/auto_ml.py

```diff
@@ -165,6 +169,19 @@ def _get_trial_hash(self, hp_dict):
         current_hyperparameters_hash = hashlib.md5(str.encode(str(hp_dict))).hexdigest()
         return current_hyperparameters_hash
 
+    def _create_logger_for_trial(self, trial_number) -> logging.Logger:
+
+        os.makedirs(self.cache_folder, exist_ok=True)
+
+        logfile_path = os.path.join(self.cache_folder, f"trial_{trial_number}.log")
+        logger_name = f"trial_{trial_number}"
+        logger = logging.getLogger(logger_name)
+        formatter = logging.Formatter("[%(asctime)s][%(levelname)s][%(module)s][%(lineno)d] : %(message)s \n", datefmt="%H:%M:%S")
```

Comment:
We're actually creating a logger for each trial, which is great when we run these trials in parallel, and overall very convenient.     Problem with what you are asking is that we need a path and a logger name when we the logger. I figured what better place to put this function then with the object that creates the trial so that the logger may be injected in the trial at creation.

Reply:
@vincent-antaki Hum, interesting. So we'd probably need logger creation functions at some point later on. It doesn't seem urgent for now.